W-14913522 - RPA Manager release notes 2024-01/2. (#333)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[january-30-2024]]
== January 30, 2024

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * Automatic RPA Bot update is supported by RPA Bot 1.5.0 and higher. RPA Manager will support the update starting with the next RPA Bot version (greater than 1.5.0).

Reviewer: ```suggestion
* RPA Manager now supports automatic RPA Bot updates for bots running version 1.5.0 or later.
+
When a new version of RPA Bot is available, RPA Manager shows a new page that enables you to update installed RPA Bots.
```

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | Run configurations can now be edited or deleted on the second and all following pages. | W-14820123

Reviewer: ```suggestion
| You can now edit and delete run configurations that show on the second and all following pages. | W-14820123
```

> | The *Deployment Map's* _Test_ tab can now be opened even if the project manager of one of the processes has been deleted. | W-14808512

Reviewer: ```suggestion
| You can now open the _Test_ tab in *Deployment Map* even if the project manager of one of the processes has been deleted. | W-14808512
```

| *Finance Analysis* no longer shows an error if the process owner of one of the processes has been deleted. | W-14820620
| *Bot Workflow Library* is now available in Japanese. | W-14836772

> | The tooltips and descriptions on the details panel of the *Bot Workflow Library* will now be shown properly. | W-14852032

Reviewer: ```suggestion
| The tooltips and descriptions on the details panel of the *Bot Workflow Library* now show properly. | W-14852032
```

RPA Manager is the cloud-based control plane for your automation projects.

> [[january-30-2024]]
> == January 30, 2024

Reviewer: ```suggestion
[[january-31-2024]]
== January 31, 2024
```

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

* RPA Manager now supports automatic RPA Bot updates for bots running version 1.5.0 or later.
+

> When a new version of RPA Bot is available, RPA Manager shows a new page that enables you to update installed RPA Bots.

Reviewer: The new *Update* button in the *Bots* view of the *Bot Management* module enables you to select the bots to update to the latest version available. 
```suggestion
When a new version of RPA Bot is available, RPA Manager shows the *Update* button in the *Bots* view of the *Bot Management* module, which enables you to update installed RPA Bots to the latest version.
```
